### DISCUSSION TITLE
custom styling the Radio buttons

### DISCUSSION THREAD

**Discussion by proton029** [Budibase/budibase#6937] [Help]
I want to change the default focus of the radio buttons in budibase which actually fills the radio button like a donut to something like this
![image](https://user-images.githubusercontent.com/88423620/181424009-a191acac-9cd6-4a38-a546-fc00dadea3f2.png)
How to do this

**Comment by kasgithub8** [comment]
Try and put this in an embed component
```
<style>
.spectrum-Radio-input.svelte-s1zsh {
}
.spectrum-Radio-input {
}
</style>
```
This is normal CSS so you can style it however you want.
